proposals/0296-Update-video-streaming-capabilities-during-ignition-cycle.md

```diff
@@ -153,30 +146,32 @@ In `VideoStreamManager` add new `startRemoteDisplayStream`:
 
 ```java
 public void startRemoteDisplayStream(Context context, Class<? extends SdlRemoteDisplay> remoteDisplayClass,
-    VideoStreamingParameters parameters, final boolean encrypted, SupportedStreamingRange streamingRange)
+    VideoStreamingParameters parameters, final boolean encrypted, VideoStreamingRange streamingRange)
 ```
 
 In this way, developers will be able to pass constraints related to specific application directly into `SDLManager`
-where `SupportedStreamingRange`:
+where `VideoStreamingRange`:
 
 ```java
-public class SupportedStreamingRange {
-    private Resolution minSupportedResolution;
-    private Resolution maxSupportedResolution;
-    private Double maxScreenDiagonal;
-    private AspectRatio aspectRatio;
+public class VideoStreamingRange {
+    private Resolution minResolution;
+    private Resolution maxResolution;
+    private Double minAspectRatio;
+    private Double maxAspectRatio;
+    private Double minScreenDiagonal;
 }
 ```
 In `SdlRemoteDisplay` implementation by mobile application developers:
 
-Later, when alternative supported resolutions are retrieved from `HMI`, `SupportedStreamingRange` will be used to unpack data:
+Later, when alternative supported resolutions are retrieved from `HMI`, `VideoStreamingRange` will be used to unpack data:
 
 ```java
 private List<VideoStreamingCapability> getSupportedCapabilities(
     Resolution minResolution,
     Resolution maxResolution,
-    Double constraintDiagonalMax,
-    AspectRatio ratioRange
+    Double minAspectRatio,
+    Double maxAspectRatio,
+    Double minScreenDiagonal
```

Comment:
These changes were not requested from the Steering Committee: https://github.com/smartdevicelink/sdl_evolution/issues/983#issuecomment-658826750.  We understand that they're incorrect, but will need a revision proposal to change, as the changes have not undergone Steering Committee review.

Reply:
@theresalech Please notice these changes have been reverted in 04adb9c200ca4f7c69d20606870be148c0d3110a
We would like to go with this update through common SDLC process as you suggested.